net/teleport/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        gravitational teleport 2.0.6 v
+revision            0
+
+homepage            http://gravitational.com/teleport/
+categories          net
+maintainers         gclawes
```

Comment:
Unless `gclawes` is your MacPorts handle and you have commit rights, you should use `@gclawes` for maintainer. Ideally please also add an obfuscated form of your email for cases when we are not able to use github mentions.

Potentially add an `openmaintainer` flag unless you want to prevent other committers to add trivial fixes to your ports.
```tcl
maintainers         {example.com:your.name @gclawes} openmaintainer
```